sdk/auth/azure_cli_authorizer.go

```diff
@@ -141,6 +144,9 @@ type azureCliConfig struct {
 
 	// AuxiliaryTenantIDs is an optional list of tenant IDs for which to obtain additional tokens
 	AuxiliaryTenantIDs []string
+
+	// SubscriptionID is the optional default subscription ID
+	SubscriptionID string
```

Comment:
calling this `DefaultSubscriptionID` would likely alleviate any confusion with this?